### PR TITLE
refactor: list again process simplified

### PR DESCRIPTION
**Pull request by AlefrankM**

### CODE REVIEW

File: src/contracts/dev/LedaNFT-address.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "address": "0xc915858E7fCe9E6cB2690fa983ba6F8D53eC75A3"
+  "address": "0x4d1b6f4D33AeC6ee506e506B7292B32f8204c74e"
```
Comment (juliswer) [inline src/contracts/dev/LedaNFT-address.json:2]: Are you sure this is the right address?

File: src/contracts/dev/Marketplace-address.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-  "address": "0xd66411A87d80793b8F7C281479780B420e396305"
+  "address": "0xB47eEa166311CaaEd0A878Cd3c167F05f89d8884"
```
Comment (juliswer) [inline src/contracts/dev/Marketplace-address.json:2]: same above

File: src/contracts/prod/Marketplace-address.json [hunk 3]
```diff
@@ -1,3 +1,3 @@
 {
-  "address": "0xC32B21a75f1763B41a591aD62b1d2fd10d5363cD"
+  "address": "0x30cAB5D4e642F03BA5d81943f6Aef3872a77A736"
```
Comment (juliswer) [inline src/contracts/prod/Marketplace-address.json:2]: same above

File: src/features/leda-nft/types/LedaNFT.ts [hunk 4]
```diff
@@ -900,6 +872,12 @@ export interface LedaNFT extends BaseContract {
       overrides?: Overrides & { from?: PromiseOrValue<string> }
     ): Promise<PopulatedTransaction>;
 
+    royaltyInfo(
+      _tokenId: PromiseOrValue<BigNumberish>,
+      _salePrice: PromiseOrValue<BigNumberish>,
+      overrides?: CallOverrides
+    ): Promise<PopulatedTransaction>;
+
```
Comment (juliswer) [inline src/features/leda-nft/types/LedaNFT.ts:880]: updated types because of the new contract, right?

**Review by juliswer — APPROVED** [review]
Looks Good!
Just make sure the addresses are right before merging main

**Review by GimerStackIt — APPROVED** [review]

File: src/contracts/dev/LedaNFT-address.json (hunk 1 above)
Comment (GimerStackIt) [reply src/contracts/dev/LedaNFT-address.json:2]: Yes, we verified the addresses.

**Pull request merged**